src/modelfree/tsne/fit_model.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import os
+import os.path as osp
+import pickle
+import re
+import tempfile
+
+import numpy as np
+import pandas as pd
+import ray
+import sacred
+from sacred.observers import FileStorageObserver
+from sklearn.manifold import TSNE
+
+from modelfree.common import utils
+
+fit_model_ex = sacred.Experiment('tsne_fit_model')
+logger = logging.getLogger('modelfree.tsne.fit_model')
+
+
+@fit_model_ex.config
+def base_config():
+    ray_server = None  # by default will launch a server
```

Comment:
Does this mean that this can't be run outside of a Ray server anymore?

Reply:
It depends on Ray but it should be seamless to the user. It just starts automatically if you don't provide a `ray_server`, and shuts it down when finished. Ray can run in a light-weight single-machine config in addition to as a cluster.